### PR TITLE
Add CVE-2020-6449

### PR DESCRIPTION
**Pull request by m-y-mo**
@xcorail This contains material linked from the chrome blog post. The link to the post in README.md is just a guess, please feel free to correct it. Thanks.

### CODE REVIEW

**Comment by xcorail** [comment]
Hey @m-y-mo 
The link is correct. I guess both PRs must be published at the same time, right?

**Comment by m-y-mo** [comment]
There isn't anything secret there, although it would probably still be nice to not leave too much a gap between the publishing times, because of broken link problems.

**Comment by xcorail** [comment]
Yep, I was just thinking about broken links. Ok, will merge tomorrow then!

File: SecurityExploits/Chrome/blink/CVE-2020-6449/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## Exploit for Chrome CVE-2020-6449
+
+The write up can be found [here](https://securitylab.github.com/research/CVE-2020-6449-exploit). This is a bug in the webaudio component I discovered in March 2020. Chromium issue ticket can be found [here.](https://bugs.chromium.org/p/chromium/issues/detail?id=1059686)
```
Comment (xcorail) [inline SecurityExploits/Chrome/blink/CVE-2020-6449/README.md:3]:
```suggestion
The write up can be found [here](https://securitylab.github.com/research/CVE-2020-6449-exploit-chrome-uaf). This is a bug in the webaudio component I discovered in March 2020. Chromium issue ticket can be found [here.](https://bugs.chromium.org/p/chromium/issues/detail?id=1059686)
```

**Review by xcorail — APPROVED** [review]

**Pull request merged**